cli/src/Morphir/Elm/CLI.elm

```diff
@@ -14,7 +14,8 @@
    limitations under the License.
 -}
 
-
+{-| This module contains definitions for ports interfacing with the CLI specified in the cli.js file
+-}
```

Comment:
Documentation of a module should come after the module declaration but before the imports. Check out this page for more details: https://package.elm-lang.org/help/documentation-format

Reply:
Noted with thanks. I'll fix this